packages/devtools_app/lib/src/logging/html_logging_screen.dart

```diff
@@ -70,6 +75,27 @@ class HtmlLoggingScreen extends HtmlScreen {
       true,
     );
 
+    final _search = (String text) {
+      controller.search(text);
+    };
+
+    final textFiled = CoreElement('input', classes: 'form-control input-sm')
+      ..setAttribute('type', 'text')
+      ..setAttribute('style', 'width:480px;')
+      ..setAttribute('placeholder', 'input text')
+      ..id = 'uri-field';
+
+    textFiled.element.onKeyDown.listen((KeyboardEvent event) {
+      if (event.charCode != null) {
+//          event.preventDefault();
+        Timer(const Duration(milliseconds: 20), () {
+          final html.InputElement inputElement = textFiled.element;
+          final String value = inputElement.value.trim();
+          _search(value);
+        });
+      }
+    });
+
```

Comment:
simplify this code to just
`_search(inputElement.value.trim());`

Reply:
rather than just setting a timer use the RateLimiter class. As is, you are still issuing the same # of requests as default but delaying the requests.

Reply:
It will lead to lose the last letter And I don't know why .